.github/workflows/docs.yml

```diff
@@ -0,0 +1,18 @@
+name: Markdown Tests
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**/*.md'
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Install Markdown linter and plugins
+        run: pnpm install remark-cli remark-lint-no-dead-urls remark-validate-links --ignore-scripts
```

Comment:
Every time we will have different versions of `remark` here.

I suggest using pnpm lock file and just use the standard step with installing Node.js and all pnpm dependencies.